cmd/root.go

```diff
@@ -135,11 +137,17 @@ var detectHelmCmd = &cobra.Command{
 	Short: "detect-helm",
 	Long:  `Detect Kubernetes apiVersions in a helm release (in cluster)`,
 	Run: func(cmd *cobra.Command, args []string) {
-		h := helm.NewHelm(helmVersion, namespace)
+		if helmStore != "secrets" && helmStore != "configmaps" && helmVersion == "2" {
```

Comment:
I like to add pre-run arg validation inside of a PreRunE when using Cobra. It is a bit cleaner IMO.

```
PreRunE: func(cmd *cobra.Command, args []string) error {
	<DO arg validation and return error if not valid>
}
```